.github/workflows/test.yml

```diff
@@ -41,7 +41,7 @@ jobs:
           key: ${{ runner.os }}-jest
 
       - name: Jest tests
-        run: NODE_OPTIONS="$NODE_OPTIONS --experimental-vm-modules" npx jest --silent --shard=${{ fromJSON(matrix.chunk) }}/${{ strategy.job-total }} --max-workers ${{ steps.cpu-cores.outputs.count }}
+        run: NODE_OPTIONS="$NODE_OPTIONS --experimental-vm-modules" npm test -- --silent --shard=${{ fromJSON(matrix.chunk) }}/${{ strategy.job-total }} --max-workers ${{ steps.cpu-cores.outputs.count }}
```

Comment:
@pasyukevich @rezkiy37 sorry can you explain how this should fix the problem we saw?

Reply:
While testing locally we noticed that the flaky tests  failing always with npx 

Moreover, locally npx that was run from the same directory is always called the higher jest version than installed in package - basically, we think this can be the root cause

Reply:
@pasyukevich ok great, can you update the PR description with this information? Can you also please update the GroupChatNameTests with the lint fixes?

Reply:
done